codalab/lib/bundle_cli.py

```diff
@@ -1201,8 +1210,21 @@ def do_upload_command(self, args):
             'metadata': metadata,
         }
 
-        # Option 1: Upload contents string
-        if args.contents is not None:
+        # Bundles are not linked by default.
+        bundle_info['metadata']['is_linked'] = False
+
+        # Option 1: --link
+        if args.link:
+            if len(args.path) is not 1:
+                raise UsageError("Only a single path can be uploaded when using --link.")
+            bundle_info['metadata']['link_url'] = args.path[0]
+            bundle_info['metadata']['is_linked'] = True
```

Comment:
That's true (it could also be inferred from `link_format`), but I thought it would clearer to have a single field that we could use to check if a bundle is linked or not.

Reply:
Also, having a separate parameter called is_linked would make it easier to filter bundles -- for example, see this part of the code (which we would likely need to filter bundles that are linked out):

https://github.com/codalab/codalab-worksheets/blob/c9169eb207d5437dd8e98dd11f983269aa32daf8/codalab/model/bundle_model.py#L2502-L2504

Reply:
I'm just going to remove is_linked for now. Let's re-add it if we find the need for it later.